dockerfiles/go-1.24.Dockerfile

```
@@ -0,0 +1,16 @@
+# syntax=docker/dockerfile:1.7-labs
+FROM golang:1.24-alpine
+
+# Ensures the container is re-built if go.mod or go.sum changes
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="go.mod,go.sum"
+
+WORKDIR /app
+
+# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
+COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)